packages/integration-karma/test/rendering/slotting/index.spec.js

```diff
@@ -0,0 +1,22 @@
+import { createElement } from 'lwc';
+
+import RenderCountParent from 'x/renderCountParent';
+
+// WIP: Need to figure out why this test doesn't pass.
```

Comment:
Add a TODO reference to https://github.com/salesforce/lwc/pull/1617 and proceed with this PR.